user/tests/tests_model.py

```diff
@@ -0,0 +1,41 @@
+# from django.test import TestCase
+from django.test import TestCase
+from user.models import User
+from django.urls import reverse, reverse_lazy
+import json
+from .tests_config import TEST_USER_CREDENTIALS as test_user
+
+# Create your tests here.
+
+
+class UserModelTest(TestCase):
+    @classmethod
+    def create_user(cls, username=test_user['username'],
+                    email=test_user['email'], first_name='event',
+                    password=test_user['password'], last_name='manager'):
+        user = User.objects.create(
+            username=username, email=email, first_name=first_name, last_name=last_name)
+        user.set_password(password)
+        user.save()
+        return user
+
+    def test_user_creation(self):
+        user_instance = self.create_user()
+        self.assertTrue(isinstance(user_instance, User))
+
+    def test_user_email(self):
+        user_instance = self.create_user()
+        # This str function will going to call the __str__() method defined in User model
+        self.assertEqual(str(user_instance), user_instance.email)
+        self.assertNotEqual(str(user_instance), 'email@email.com')
```

Comment:
@usman-hussain we werent doing these kinds of tests?

Reply:
@mashhoodr Actually this str() is calling the __str__() method defined in user model which is returning user email. So, this is particularly to test the __str__() method defined in model.

Reply:
@usman-hussain I dont think this is useful, its not really testing any logic?

Reply:
@mashhoodr Alright I'll remove it.